File: app/src/main/java/com/akexorcist/localizationapp/viewpager/OneFragment.java
package com.akexorcist.localizationapp.viewpager;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.akexorcist.localizationapp.R;

/**
 * Created by Akexorcist on 7/22/15 AD.
 */
public class OneFragment extends Fragment {

    public static Fragment newInstance() {
        return new OneFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_one, container, false);
    }
}

File: app/src/main/java/com/akexorcist/localizationapp/viewpager/OneFragment.kt
package com.akexorcist.localizationapp.viewpager

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.akexorcist.localizationapp.databinding.FragmentViewPagerOneBinding

class OneFragment : Fragment() {
    companion object {
        @JvmStatic
        fun newInstance(): Fragment = OneFragment()
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        val binding = FragmentViewPagerOneBinding.inflate(layoutInflater, container, false)
        return binding.root
    }
}
